vtr_flow/scripts/python_libs/vtr/flow.py

```diff
@@ -167,6 +168,15 @@ def run(
     shutil.copy(str(circuit_file), str(circuit_copy))
     shutil.copy(str(architecture_file), str(architecture_copy))
 
+    # Extract includes path
```

Comment:
This comment doesn't seem to describe the whole block of code (does more than finding the include path). Describe fully what this block of code does.